Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+  agent {
+    label 'sbt-slave'
+  }
+
+  environment {
+    // Ensure that build scripts recognise the environment they are running within
+    CI = 'jenkins'
+  }
+
+  stages {
+    stage('Compile') {
+      steps {
+        ansiColor('xterm') {
+          sh "sbt clean compile doc"
```

Comment:
`;` **before** a command schedules all tasks to the sbt graph scheduler (i.e. in parallel).